auacm/app/modules/competition_manager/views.py

```diff
@@ -181,3 +202,129 @@ def get_competition_data(cid):
         'compProblems': comp_problems,
         'teams': scoreboard
     })
+
+
+@app.route('/api/competitions/<int:cid>/register', methods=['POST'])
+@login_required
+def register_for_competition(cid):
+    """ Called when a user wants to register for a competition.
+
+    All the user has to do is submit a post to this url with no form data.
+    From their logged-in status, we'll go ahead and add them to the competiton
+    as an individual (team name is default their display name).
+    """
+    if session.query(Competition).filter(Competition.cid == cid).first() \
+            is None:
+        return serve_error('Competition does not exist', response_code=404)
+
+    if current_user.admin == 1 and 'users' in request.data:
+        registrants = loads(request.data['users'])
+    else:
+        registrants = list()
+        registrants.append(current_user.username)
+
+    for user in registrants:
+        if session.query(CompUser).filter(CompUser.cid == cid,\
```

Comment:
I'm not going to make you change it (I know we've done it elsewhere), but Google style [forbids backslash continuation.](https://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length)

I'm not sure it's 100% necessary in the places you used it, particularly around [parens, brackets, and braces](http://docs.python.org/reference/lexical_analysis.html#implicit-line-joining).